tool/benchtool/BenchTool.py

```diff
@@ -161,6 +190,10 @@ def __apply_variant_in_impl(self, workload: Entry, variant: Variant) -> None:
         '''
         Helper for applying variant.
         '''
+        # Save base implementation.
+        if variant.filename not in self.__original:
```

Comment:
Does this assume that the first variant is the base implementation? Sometimes we skip the base implementation (to save time). What happens then?